src/sentry/api/bases/organizationmember.py

```diff
@@ -2,14 +2,14 @@
 
 from typing import Any
 
-from django.db.models import Q
 from rest_framework import serializers
 from rest_framework.request import Request
 
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.db.models.fields.bounded import BoundedAutoField
 from sentry.models import InviteStatus, Organization, OrganizationMember
 
+from ...services.hybrid_cloud.user.service import user_service
```

Comment:
```suggestion
from sentry.services.hybrid_cloud.user.service import user_service
```

Relative imports are :cry: